tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -689,3 +693,36 @@ def test_positive_overview_label_name(form_data, session):
             assert results['overview'][key] == value
         session.virtwho_configure.delete(name)
         assert not session.virtwho_configure.search(name)
+
+
+def test_positive_last_checkin_status(form_data, session):
+    """Verify the Last Checkin status on Content Hosts Page.
+
+    :id: 3931e10e-8adc-4ca4-8963-20572b2f99bf
+
+    :expectedresults: The Last Checkin time on Content Hosts Page is UTC time
+
+    :BZ: 1652323
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        values = session.virtwho_configure.read(name)
+        command = values['deploy']['command']
```

Comment:
Since you are not using `values` (except for extracting `command`), you should be able to get away with reading only this one widget. Should speed up test a little.

```python
value = session.virtwho_configure.read(name, widget_names='deploy.command')
command = value['deploy']['command']
```
(you could do that in one line, but it would be ugly)

Reply:
ok, thanks